docs/how-to-guides/develop-a-flow/develop-conditional-flow.md

```diff
@@ -0,0 +1,69 @@
+# Develop conditional flow
```

Comment:
@PeiwenGaoMS this is not a good place to put the doc.
The problem is that this adds confusion "conditional flow" is a parallel concept with "standard flow". It's not.

I would rename the doc to be "Add conditional control to your flow" and put it under the root folder of "how-to-guides".

On a side note - @wangchao1230 why don't we have develop-chat-flow and develop-evaluation-flow under the folder "develop-a-flow"?

Reply:
we have the develop-chat-flow in this PR: https://github.com/microsoft/promptflow/pull/554
for develop evaluation flow, I created a task on this.
[Task 2703094](https://msdata.visualstudio.com/Vienna/_workitems/edit/2703094): Add doc: how to develop evaluate flow

Reply:
Updated.